.ruby-version

```diff
@@ -1 +1 @@
-2.7.2
+3.1.2
```

Comment:
While we still test in CI on 2.7, I figure it's time to bump this while we're at it.

Maybe some local scripts will break? But we can address those as they happen.